sdk/src/poh_config.rs

```diff
@@ -3,7 +3,7 @@ use {
     std::time::Duration,
 };
 
-#[derive(Serialize, Deserialize, Clone, Debug, AbiExample, Eq, PartialEq)]
+#[derive(Serialize, Deserialize, Clone, Copy, Debug, AbiExample, Eq, PartialEq)]
```

Comment:
`PohConfig` is a little large for `Copy`.  Prefer passing them around by reference and `clone()`ing where we need to keep one.